src/ajax/interceptors.cljc

```diff
@@ -170,26 +170,31 @@
                    headers))))
   (-process-response [_ xhrio] xhrio))
 
-(m/defn-curried ^:internal uri-with-params [{:keys [vec-strategy params]} uri]
+(m/defn-curried ^:internal uri-with-params [{:keys [vec-strategy params method url-params]} uri]
   "Internal function. Takes a uri and appends the interpretation of the query string to it
    matching the behaviour of `url-request-format`."
-  (if params
```

Comment:
Couple of observations here: 

* do we want `:params` to take priority of `:url-params`. Since existing code won't have both parameters, backwards compatibility isn't an issue.
* Rather than copy and pasting, could we turn the if statement into an if-let, that assigns to "params or url params" (needs a better variable name)

Reply:
Changed this to an if-let with a conditional binding on whether this is a get or not.

Valid point about the priority. I changed it such that if it's a GET, and `:url-params' are nil, it defaults to using `:params'. Otherwise it will always use `:url-params`.  I also added a test to ensure the priority.